test/selenium/specs/ts_all_specs.rb

```diff
@@ -4,4 +4,7 @@
 require_relative "tc_show_injected"
 require_relative "tc_extension_options"
 require_relative "tc_history"
+if @browser == "firefox"
+  require_relative "tc_posting_process"
```

Comment:
The tests won't pass for Chrome(because the posting process has changed) and Safari(I just noticed there's a PR open for the posting button).

Should I remove it ?